src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1449,6 +1449,8 @@ static int CheckEventJSON(cJSON *event, cJSON **scan_id, cJSON **id, cJSON **nam
 
         if ( *result = cJSON_GetObjectItem(*check, "result"), !*result) {
             *result = cJSON_CreateString("not applicable");
+            cJSON_AddItemToObject(*check, "result", *result);
+            cJSON_ReplaceItemInObject(event, "check", *check);
```

Comment:
I think this call has no effect.

Reply:
Done https://github.com/wazuh/wazuh/pull/16214/commits/5f73c4022232e6d509b9083abd933589876298fe and tested again:

```
==7759== LEAK SUMMARY:
==7759==    definitely lost: 0 bytes in 0 blocks
==7759==    indirectly lost: 0 bytes in 0 blocks
==7759==      possibly lost: 6,336 bytes in 22 blocks
==7759==    still reachable: 23,628,990 bytes in 128,082 blocks
==7759==         suppressed: 0 bytes in 0 blocks
==7759== Reachable blocks (those to which a pointer was found) are not shown.
==7759== To see them, rerun with: --leak-check=full --show-leak-kinds=all
```